### ISSUE TITLE
Character Not saving & Loading Properly

### ISSUE THREAD

**Issue by doomsguy214** [Kyle873/DoomRPG#100]
I have running RL Arsenal plus Doom RPG from https://github.com/Kyle873/DoomRPG/archive/86c6ba36d081a3c8228f812374b53cb735c2e53c.zip

It runs fine. But every try to directly go to outpost & save my character my weapons disapper. When i load my character its say save corrupted error 3

**Comment by Sumwunn** [comment]
I found that version to be pretty buggy. 
I had the scripts crash after completing a mission at about 10 maps in. Forcing me to restart.

I noticed the DoomRPG.lib from that release is from October while the one from the here is from March. So I replaced it with the newer one here and so far I'm having no issues. I'm 15 maps in and have done plenty of missions. Using GZDoom v3.0.0.

Hope this helps.

**Comment by dongmaculate** [comment]
Created account for this comment. Supplanting that file into the above stable release just causes these errors immediately before failing to launch.

*Script error, ":actors/doomrl/powerups.txt" line 32:*
*Parent type 'RLInvulnerabilitySpherePickupGiver' not found in RLInvulnerabilitySpherePickupGiver2*

*Script error, ":actors/doomrl/powerups.txt" line 32:*
*Replaced type 'RLInvulnerabilitySpherePickupGiver' not found for RLInvulnerabilitySpherePickupGiver2*

*Script error, ":actors/doomrl/powerups.txt" line 34:*
*"INVENTORY.AUTOACTIVATE" is an unknown flag*

*Script error, ":actors/doomrl/powerups.txt" line 35:*
*"INVENTORY.ADDITIVETIME" is an unknown flag*

*Script error, ":actors/doomrl/powerups.txt" line 37:*
*'powerup.color' requires an actor of type 'Inventory'*

*Execution could not continue.*

*Script error, ":actors/doomrl/powerups.txt" line 37:*
*Unexpected '255' in definition of 'RLInvulnerabilitySpherePickupGiver2'*

To my knowledge I'm running the most recent available versions of RLA and DRPG along with GZDoom 3.2.0. Was the most recent version of the library expecting the version of RLA that isn't out yet?

**Comment by worldendDominator** [comment]
Yes, fresh DRPG doesn't work with current DRLA. You need to get an older version linked in the ZDoom forum topic.

**Comment by dongmaculate** [comment]
Aye, which is what this issue is related to to begin with. Back to OP, attempting to save a character and reload it later causes the aforementioned Error code 3. I think it may have something to do with the locker and the number of items it attempts to save?

**Comment by rastamangames** [comment]
By the way, where exactly do character data stored?
I need it in case of sweaty backup.
